### PR TITLE
docs(http): add information about body content type to flush method for testing

### PR DESCRIPTION
**Pull request by web-dave**
run into this.
hope this helps.

### CODE REVIEW

File: packages/common/http/testing/src/request.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class TestRequest {
   /**
    * Resolve the request by returning a body plus additional HTTP information (such as response
    * headers) if provided.
+   * The body is converted to requested contentType. Keep in mind: HttpClient set contentType to `json` by default.
```
Comment (brandonroberts) [inline packages/common/http/testing/src/request.ts]: "If provided, the body is converted to the requested contentType. The body is converted to `json` by default."

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Update git commit message to `docs(http): add information about body content type to flush method for testing`

File: packages/common/http/testing/src/request.ts (hunk 1 above)
Comment (web-dave) [reply packages/common/http/testing/src/request.ts]: thx, I'll fix this. Sorry, no native speaker.

Comment (brandonroberts) [reply packages/common/http/testing/src/request.ts]: No problem 👍

Comment (web-dave) [reply packages/common/http/testing/src/request.ts]: Done. My first PR to the Framework (ok, just testing comment)

**Review by sreevani-ship-it — COMMENTED** [review]
Re-write needed. Suggestion: 
Resolve the request by returning a body plus additional HTTP information (such as response headers) if provided.
If a contentType is provided, the body is converted to the requested type. Otherwise, the body is converted to JSON by default.

**Comment by brandonroberts** [comment]
@web-dave you'll also need to run the code formatter over your changes. From the root of the project.

```sh
yarn
```

```sh
yarn gulp format
```

**Comment by web-dave** [comment]
lot of effort, just to add a comment. But thats how this things work.

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: This is a docs only change and should not impact g3. Please run presubmit.

**Comment by kara** [comment]
Still needs review from fw-http, so removing merge assistance for now

File: packages/common/http/testing/src/request.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ export class TestRequest {
   /**
    * Resolve the request by returning a body plus additional HTTP information (such as response
    * headers) if provided.
+   * If a contentType is provided, the body is converted to the requested type.
```
Comment (alxhub) [inline packages/common/http/testing/src/request.ts]: I don't think the wording is quite correct here. It's not the `contentType` of the response, but the request's `responseType` which comes from the original request's `response` type narrowing.

Maybe
> If the request specifies an expected body type, the body is converted into the requested type.

**Review by alxhub — CHANGES_REQUESTED** [review]

**Review by alxhub — APPROVED** [review]

**Pull request closed without merging**